pyproject.toml

```diff
@@ -8,15 +8,15 @@ description = "API for questionnaire schema validation"
 authors = ["ONSDigital"]
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.11"
 Flask = "^2.0.2"
 jsonpointer = "^2.2"
 structlog = "^21.5.0"
 python-dateutil = "^2.8.2"
 jsonschema = "4.4.0"
 jsonpath-rw = "^1.4.0"
 jsonpath-rw-ext = "^1.2.2"
-eq-translations = {git = "https://github.com/ONSDigital/eq-translations.git", rev = "v4.9.0"}
+eq-translations = {git = "https://github.com/ONSDigital/eq-translations.git", rev = "v4.9.1"}
```

Comment:
While we are here, we should update these versions to the latest available. Useful plugin: https://github.com/MousaZeidBaker/poetry-plugin-up

Reply:
We should update to latest available, not latest within the version constraints.

Reply:
Pretty sure thats what I have done, I used `poetry up --latest`

Reply:
Right, I think I may have been looking at the first commits which wasn't. All good then 👍

Reply:
Ah yeah, I ran it without `--latest` at first 👍